notebooks/official/workbench/exploratory_data_analysis/explore_data_in_bigquery_with_workbench.ipynb

```diff
@@ -1060,7 +985,7 @@
         "\n",
         "*   The United States is the country with the most events, followed by India and the United Kingdom.\n",
         "*   \"Web\" is the most common device operating system by far, with iOS and Windows pretty close for second. \n",
-        "*   The most common value in the device language field is actually NULL, or no data in the field. This is important to note in any sort of further analysis or modeling using language. You may want to understand why it's missing, what types of users/events have missing language more or less often, and how that may/may not change in future data."
+        "*   The most common value in the device language field is actually NULL, or no data in the field. This is important to note in any sort of further analysis or modeling using language. You may want to understand why it's missing, what types of users/events have missing language more or less often, and how that might change in future data."
```

Comment:
Again, clarify what "it" is referencing ("it's missing...")

Reply:
it referring to -->  language field , as it is null or no data in the field